libs/data-mapper/src/lib/mapDefinitions/__test__/MapDefinitionDeserializer.spec.ts

```diff
@@ -579,7 +579,7 @@ describe('mapDefinitions/MapDefinitionDeserializer', () => {
       );
     });
 
-    it('creates a direct index connection', () => {
+    it('creates a direct access connection', () => {
```

Comment:
Nit: this is actually specifically for custom value index access instead of using the index function. Maybe `creates a custom value direct access connection`?